service/src/main/java/com/fecklessweasel/service/objectmodel/StoredFile.java

```diff
@@ -296,17 +296,20 @@ public String getDescription() {
     /**
      * Look up all files associated with a specific course ID.
      * @param sql The Sql connection to the FecklessWeaselDB
-     * @param cid The ID of the course whose file info we want.
+     * @param course The the course whose file info we want.
      * @return A List of StoredFile objects that represent each files info.
      */
-    static Iterable<StoredFile> lookUpCourseFiles(Connection sql, int cid)
+    public static Iterable<StoredFile> lookupCourseFiles(Connection sql, Course course)
```

Comment:
@et8109 i think this follows the current pattern, which is:

*Implement the lookup in stored file (since StoredFile.java is the only class that can access the StoredFile constructor)

*Then implementing the method in Courses that calls the lookup method, but passes in this.cid, so we can lookup all files that belong to the course